tmplctlr/metrics.go

```diff
@@ -0,0 +1,16 @@
+package tmplctlr
```

Comment:
This is the same gauge we would want to use no matter what the controller was isn't it? If so shouldn't this be defined outside the controller and passed into the controller?